superset-frontend/src/views/CRUD/welcome/ActivityTableRow.tsx

```diff
@@ -83,7 +83,7 @@ const ActivityTableRow: React.FC<ActivityProps> = ({
   <>
     {activityList.map(activity => (
       <CardStyles
-        key={`${activity.item_title}__${activity.time}`}
+        key={`${activity.dashboard_title}__${activity.time}__${activity.id}`}
```

Comment:
This solved the problem of repeated keys.

Reply:
@ktmud  The "infinite cards bug" is a problem with keys (or lack thereof).

Reply:
These types need a better definition. Several "mandatory" fields are present only sometimes (example, id) ...
This definition is not at all reliable...

Reply:
If `id` is always present, then `dashboard_title` is probably not needed. Feel free to update the typing based on API responses.

Reply:
@ktmud That's the problem, the `id` is **not always present**. On the main tab page it does not exist. In the other tabs it exists (with the test data). This typing is not reliable.

Reply:
That's unfortunate and probably another reason why we want to do the typing refactoring first. It really isn't that hard to do because most of the types already exist somewhere else. See: https://github.com/apache/superset/pull/13291